common/src/main/java/org/popcraft/chunkyborder/BorderCheckTask.java

```diff
@@ -173,4 +189,28 @@ private boolean wrapEarth(final BorderData borderData, final Location location)
         }
         return true;
     }
+
+    // pretend this part doesn't exist, will be replaced by a proper method call once possible
+    private static final java.lang.invoke.MethodHandle GET_ELEVATION_AT_ASYNC;
+
+    static {
+        java.lang.invoke.MethodHandle temp;
+
+        try {
+            temp = java.lang.invoke.MethodHandles.publicLookup().unreflect(World.class.getMethod("getElevationAtAsync", int.class, int.class));
+        } catch (ReflectiveOperationException e) {
+            throw new RuntimeException(e);
+        }
+
+        GET_ELEVATION_AT_ASYNC = temp;
+    }
+
+    @SuppressWarnings("unchecked")
+    private CompletableFuture<Integer> getElevation(final World world, final int x, final int z) {
+        try {
+            return (CompletableFuture<Integer>) GET_ELEVATION_AT_ASYNC.invokeExact(world, x, z);
+        } catch (Throwable e) {
+            return CompletableFuture.failedFuture(e);
+        }
+    }
```

Comment:
As a tip, you can publish chunky to maven local and then you should also be able to build locally against it.

Target chunky version here will need a bump since this is a breaking change (but after we merge it into chunky so we have the actual final version number). This will prevent CB from loading on older versions of Chunky, requiring an update.
https://github.com/pop4959/ChunkyBorder/blob/37690f66dd45a618d6c2556dc606d1fa99c15ab2/gradle.properties#L5